docs/src/app/(docs)/react/components/checkbox/page.mdx

```diff
@@ -28,11 +28,26 @@ import { Checkbox } from '@base-ui/react/checkbox';
 
 ## Examples
 
+### Labeling a checkbox
+
+Label checkboxes using an enclosing `<label>` element that wraps `<Checkbox.Root>`. Enclosing labels ensure that clicking on any gaps between the label and checkbox still toggles the control, and announce the label correctly to screen readers.
+
+The [Field](/react/components/field) component can also be used to provide accessible labels for individual checkbox without needing to keep track of manual `id` or `aria-label` associations. It supports both enclosing and sibling labeling patterns, along with a UX improvement to prevent double clicks from selecting the label text.
```

Comment:
Grammarly suggestion.
```suggestion
The [Field](/react/components/field) component can also be used to provide accessible labels for individual checkboxes, eliminating the need to track `id` or `aria-label` associations manually. It supports both enclosing and sibling labeling patterns, along with a UX improvement to prevent double clicks from selecting the label text.
```